tests/test_db_models.py

```diff
@@ -12,12 +12,6 @@ def setUp(self):
         db.drop_all()
         db.create_all()
 
-        # add a seller row
-        self.EMAIL = "KidsKrafts4U@gmail.com"
-        new_seller = Seller(email = self.EMAIL, password = '1234')
-        db.session.add(new_seller)
-        db.session.commit()
```

Comment:
@ethanschreur why are we removing this ?

Reply:
> @ethanschreur why are we removing this ?

Since we are hardcoding the login credentials into the application, there is no need for a sellers table.

Reply:
Ok ....That is fine.